docker/generic/start_proxy.py

```diff
@@ -135,6 +135,28 @@ def make_argparser():
         manually. Valid schemes are `http`, `https`, `grpc`, and `grpcs`.
         '''.format(backend=DEFAULT_BACKEND))
 
+    parser.add_argument(
+        '--enable_backend_address_override',
+        action='store_true',
+        help='''
+        Backend addresses can be specified using either the --backend flag
+        or the `backend.rule.address` field in the service configuration.
```

Comment:
also mention x-google-backend address in openapi spec.

Reply:
OK, briefly mentioned it for usability.

In general, let's not mention OpenAPI too much. ESPv2 should not care about OpenAPI spec. The input is service config. Also the fact that `backend.rules.address` is set by `x-google-backend` is implementation detail not related to ESPv2.